.github/workflows/build_rpms.yaml

```diff
@@ -0,0 +1,55 @@
+name: Build RPMs
+
+on:
+  # push:
+  schedule:
```

Comment:
we probably also want workflow_dispatch so we can build manually on demand

Reply:
Done